### DISCUSSION TITLE
%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%985/

### DISCUSSION THREAD

**Discussion by giscus** [SungbinYang/SungbinYang.github.io#9] [Comments]
# %EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%985/

양성빈의 블로그

https://sungbin.kr/%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%985/

**Comment by moonnahye** [comment]
아 공백을 저렇게 적으면 간단해지는군요!  코드 참고해서 다시 수정해보겠습니당. 변수이름도 row라고 하니까 더 읽기 쉬운거같아요 저도 고쳐보겠습니다 !!
